salt/docker/init.sls

```diff
@@ -23,10 +23,10 @@ include:
 #######################
 
 {% set docker_opts = salt['pillar.get']('docker:args', '')%}
-{% set docker_reg  = salt['pillar.get']('docker:registry', '') %}
-{% if docker_reg|length > 0 %}
-  {% set docker_opts = docker_opts + " --insecure-registry={{ docker_reg }} --registry-mirror=http://{{ docker_reg }}" %}
-{% endif %}
+{% set docker_regs  = salt['pillar.get']('docker:registry', '').split(',') %}
+{% for reg in docker_regs if docker_regs|length > 0 %}
+  {% set docker_opts = docker_opts + " --insecure-registry={{ reg }} --registry-mirror=http://{{ reg }}" %}
```

Comment:
Registries need the `--insecure-registry` if there is no trusted certificate for them.